### DISCUSSION TITLE
DEVs/Community contact form/channel

### DISCUSSION THREAD

**Discussion by 0xsingular** [pkgxdev/pkgx#885] [Ideas]
**Proposal**
A contact form for developers and/or other categories of proposals needs to be created.
A channel on discord that will redirect developers/community to the appropriate repository on GitHub will be more than perfect.  
This idea will be more necessary to realize once the testnet is launched, for a more effective organization in the DEVs/community collaboration with the tea.xyz team.

**Comment by mfts** [comment]
Thanks for bringing up new ideas, @0xsingular 👍

The team is thinking about some of these challenges already. 

Please feel encouraged to share more ideas here or on discord.

**Reply by 0xsingular** [reply]
Glad the team is thinking about it, [discord forum channel](https://support.discord.com/hc/en-us/articles/6208479917079-Forum-Channels-FAQ) can be a good option for communication during testnet, the discussion will be organized and devs will be able to follow and engage in specific issues.

**Reply by mfts** [reply]
I'll take that into consideration. Thanks :)

**Comment by adam50200** [comment]
Idea not good
